proto/executor.proto

```diff
@@ -78,3 +78,11 @@ message Limit {
 	// Limit the result to be returned.
 	optional uint64 limit = 1 [(gogoproto.nullable) = false];
 }
+
+message ExecutorExecutionSummary {
```

Comment:
we also need the information about "how many times the executor.next function is called."